### PR TITLE
Converted test cases

### PR DESCRIPTION
**Pull request by ksr30**

### CODE REVIEW

**Comment by ksr30** [comment]
[DEX-202.11 [DEV] Fix Unit tests & Run tests](https://trello.com/c/Hd8qnK8e/52-dex-20211-dev-fix-unit-tests-run-tests)

File: src/tests.rs [hunk 1]
```diff
@@ -9,7 +9,7 @@ use sp_core::H256;
 
 use super::*;
 
-const UNIT: u128 = 1000000000000000000;
+const UNIT: u128 = 1_000_000_000_000;
 
 fn setup_native() {
     let native_asset =  ("Native").using_encoded(<Test as frame_system::Trait>::Hashing::hash);
```
Comment (Gauthamastro) [inline src/tests.rs:15]: The total issuance should be the sum of all tokens in the system, so if Alice has a free balance of 100, reserve balance 4, and frozen 2, then total_issuance is 106. 

The same problem is there for setup_asset_pallet()

File: src/tests.rs [hunk 2]
```diff
@@ -84,60 +84,60 @@ fn test_currency_trait_implementation () {
...
-        assert_eq!(Native::burn(50u128), PositiveImbalance::new(50u128*UNIT));
+        assert_eq!(Native::burn(50u128*UNIT), PositiveImbalance::new(50u128*UNIT));
     });
 
     // Burn amount more then total issuance
     new_test_ext().execute_with(|| {
         setup_native();
         let alice: u64 = 1;
-        assert_eq!(Native::burn(250u128), PositiveImbalance::new(200u128*UNIT)); //@Gautham check this
+        assert_eq!(Native::burn(250u128*UNIT), PositiveImbalance::new(200u128*UNIT)); //@Gautham check this
```
Comment (Gauthamastro) [inline src/tests.rs:94]: It is correct! But asset total_issuance after burn

File: src/tests.rs [hunk 3]
```diff
@@ -84,60 +84,60 @@ fn test_currency_trait_implementation () {
...
     });
 
     // Issue amount
     new_test_ext().execute_with(|| {
         setup_native();
         let alice: u64 = 1;
-        assert_eq!(Native::issue(250u128), NegativeImbalance::new(250u128*UNIT));
+        assert_eq!(Native::issue(250u128*UNIT), NegativeImbalance::new(250u128*UNIT));
```
Comment (Gauthamastro) [inline src/tests.rs:101]: Add assert for total_balance after issue. It should be increased by 250

File: src/tests.rs [hunk 4]
```diff
@@ -84,60 +84,60 @@ fn test_currency_trait_implementation () {
...
     });
 
     // Test Slash - Slashing amount less then Free Balance
     new_test_ext().execute_with(|| {
         setup_native();
         let alice: u64 = 1;
-        assert_eq!(Native::slash(&alice, 50u128), (NegativeImbalance::new(50u128*UNIT), 0u128*UNIT));
+        assert_eq!(Native::slash(&alice, 50u128*UNIT), (NegativeImbalance::new(50u128*UNIT), 0u128*UNIT));
     });
```
Comment (Gauthamastro) [inline src/tests.rs:109]: Assert user free_balance and reserve balances after the slash

File: src/tests.rs [hunk 5]
```diff
@@ -84,60 +84,60 @@ fn test_currency_trait_implementation () {
...
         setup_native();
         let alice: u64 = 1;
-        assert_eq!(Native::slash(&alice, 104u128), (NegativeImbalance::new(104u128*UNIT), 0u128*UNIT));
+        assert_eq!(Native::slash(&alice, 104u128*UNIT), (NegativeImbalance::new(104u128*UNIT), 0u128*UNIT));
     });
 
     // Test Deposit Creating
     new_test_ext().execute_with(|| {
         setup_native();
         let alice: u64 = 1;
```
Comment (Gauthamastro) [inline src/tests.rs:128]: Assert user balance after the action for these functions

**Review by Gauthamastro — CHANGES_REQUESTED** [review]
Some more asserts of user balances are required before merging

**Comment by ksr30** [comment]
Done @Gauthamastro

**Review by Gauthamastro — APPROVED** [review]
All looks good for now. We will come back and check if anything breaks later.

**Pull request merged**